frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/runLogs/LogsTabStatus.tsx

```diff
@@ -38,13 +38,13 @@ const LogsTabStatus: React.FC<LogsTabStatusProps> = ({ error, loaded, refresh, o
         latest {LOG_TAIL_LINES} lines. Exceptionally long lines are abridged. To view the full log
         for this task, you can{' '}
         <Button
-          isDisabled={!!onDownload}
+          isDisabled={!onDownload}
```

Comment:
As we discussed... the alternative was to make `onDownload` required.

I think this is fine, but worth noting, the dependency being optional is why we have to handle this, and the function is always available and passed in without issue.